pkgs/applications/graphics/vimiv-qt/default.nix

```diff
@@ -18,7 +18,7 @@ python3.pkgs.buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles qt5.wrapQtAppsHook python3.pkgs.setuptools ];
 
-  propagatedBuildInputs = with python3.pkgs; [ pyqt5 py3exiv2 qt5.qtsvg ];
+  propagatedBuildInputs = with python3.pkgs; [ pyqt5 py3exiv2 qt5.qtsvg ] ++ (with pkgs; [ qt5.qtwayland ]);
```

Comment:
there's no pkgs in scope
qtwayland shouldn't be propagated

Reply:
qtsvg also shouldn't be in propagated

```
buildInputs = [
  qtsvg
] ++ lib.optionals stdenv.isLinux [
  qtwayland
];
```
qtwayland is behind a isLinux because macos doesn't need or have qtwayland and this package has platforms = all

Reply:
Sorry, I don't really understand what you mean, is it a "style" issue? (Because it works like this)

Reply:
propagatedBuildInputs vs buildInputs isn't a style issue. Yes it may work but it's still unneeded propagation.